modules/osdk-multi-arch-building-images.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-multi-arch-support.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-multi-arch-building-images_{context}"]
+= Building a manifest list of the platforms your Operator supports
+
+You can use the `make docker-buildx` command to build a manifest list of the platforms supported by your Operator and operands. A manifest list points to specific image manifests for one or more architectures. An image manifest specifies the platforms that an image supports.
```

Comment:
Feel free to correct me if I'm misinterpreting the question. The way I'm interpreting this question is "Is the Operator-SDK opinionated on how operand images  (i.e images used in a Deployment created by the operator) are built?".

Short answer is no. I think a pretty common pattern for operators is deploying some form of an application, an example being the Operator-SDK tutorial of building an operator for deploying memcached. In the example, the creation of a `Memcached` CR results in a deployment being created that uses some other OCI image for memcached. The key here is that the Operator and the application are two separate things and it is assumed the application has it's own process for building images for the architectures supported (typically some variation of `docker buildx`).  The step in the prerequisites a bit further down is one that makes you aware of what architectures the application image itself supports and that helps you in deciding what architectures your operator should support, with the common case being to support the same architectures the operand image(s) support.

For an operator that doesn't have any operands that end up being workload type resources (think something like an operator that creates RBAC, secrets, configmaps, etc.) it is up to the operator authors to evaluate what architectures it is likely their user's nodes will be running and build for those architectures.

Reply:
@michaelryanpeter I think the key thing here is - do the docs focus on explaining who to build and distribute an operator or both the operator and its operands?

It seems like a slippery slope to try to explain how different applications should be built, since its going to different per application. That said, we should should have a note somewhere that says something like, hey, if you're building the operator for multiple architectures, your application should probably support them too. You can find more support for building containers for multiple arches here {Insert-link-to-kubebuilder}.

Reply:
I will work on that note. Unfortunately, we have rules from support about what kinds upstream documentation we are allowed to link to. Basically, we don't link to upstream procedures, since their support status is unclear. I will try to think through that a bit.

Reply:
@everettraven and @jaypoulz: Do you think[ this admonition](https://github.com/openshift/openshift-docs/pull/64355/files#diff-4acb0b31c4e1bd9042a81b7a8fa87ebf61d412bfb9100d182acb7af42277a3b0R13-R16) is enough for now?

Reply:
Alongside https://github.com/openshift/openshift-docs/pull/64355/files#diff-4acb0b31c4e1bd9042a81b7a8fa87ebf61d412bfb9100d182acb7af42277a3b0R33 I think this covers it. :)

Reply:
+1, I think that is sufficient